### PR TITLE
Add kubectl and kustomize netlify sites for cli-experimental

### PR DESCRIPTION
**Pull request by pwittrock**

### CODE REVIEW

File: dns/zone-configs/k8s.io._0_base.yaml [hunk 1]
```diff
@@ -299,3 +299,13 @@ krew.sigs:
 secrets-store-csi-driver.sigs:
 - type: CNAME
   value: kubernetes-sigs-secrets-store-csi-driver.netlify.app.
+# https://github.com/kubernetes-sigs/cli-experimental docs (@pwittrock @seans3 @eddiezane @soltysh)
+# kustomize and kubectl share the same docs site
+kustomize.sigs:
+  type: CNAME
```
Comment (munnerz) [inline dns/zone-configs/k8s.io._0_base.yaml:305]: I think this is meant to have a - before it as it is a list of structures (like the entry above)

Comment (munnerz) [reply dns/zone-configs/k8s.io._0_base.yaml:305]: Ah, I see your discussion on slack now and that other entries do not 🙈

Comment (pwittrock) [reply dns/zone-configs/k8s.io._0_base.yaml:305]: :)

**Comment by munnerz** [comment]
@pwittrock seems like the netlify site needs to be properly configured to fetch a certificate that's valid for these new DNS names 😊 https://kubectl.sigs.k8s.io

Also, does one of these two domain names redirect to the other? Else we'll end out with duplicate google search entries/pagerank profiles

**Comment by pwittrock** [comment]
@munnerz looks like it is fixed now

**Pull request merged**